### PR TITLE
Adding coverage to documents.py. First contribution to toga! :)

### PR DESCRIPTION
**Pull request by tiporan**
## PR Checklist:

- [x] All new features have been tested
- [x] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Comment by danyeaw** [comment]
@tiporan Thanks for the contribution! To start with, could you please checkoff the PR checklist above?

File: src/core/tests/test_documents.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        self.document_type = 'path/to/document.txt'
+        self.document = toga.Document(filename=self.filename,
+                                   document_type=self.document_type,
+                                   app=toga_dummy)
+
+    def test_app(self):
+        self.assertEqual(self.filename, self.document.filename)
+
+    def test_read(self):
+        with self.assertRaises(NotImplementedError): self.document.read()
```
Comment (Aareon) [inline src/core/tests/test_documents.py]: No newline at EOF

Comment (freakboy3742) [reply src/core/tests/test_documents.py]: A much bigger concern than the newline is wrapping the context manager onto a single line. 

This *should* have been picked up by our CI service - we run a bunch of automated code format checks on every pull request, but for some reason, they didn't fire on this PR. Don't worry - it wasn't anything *you* did, @tiporan, but once I resolve the problem, you'll probably get a bunch of notifications about code format problems from @brutusthebee.

File: src/core/tests/test_documents.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class DocumentTests(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.filename = 'path/to/document.txt'
+        self.document_type = 'path/to/document.txt'
+        self.document = toga.Document(filename=self.filename,
+                                   document_type=self.document_type,
+                                   app=toga_dummy)
```
Comment (brutusthebee) [inline src/core/tests/test_documents.py]: At column 36: (E128) continuation line under-indented for visual indent

File: src/core/tests/test_documents.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (brutusthebee) [inline src/core/tests/test_documents.py]: At column 36: (E128) continuation line under-indented for visual indent

File: src/core/tests/test_documents.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    def test_app(self):
+        self.assertEqual(self.filename, self.document.filename)
+
+    def test_read(self):
+        with self.assertRaises(NotImplementedError): self.document.read()
\ No newline at end of file
```
Comment (brutusthebee) [inline src/core/tests/test_documents.py]: At column 1: (W391) blank line at end of file

Comment (brutusthebee) [inline src/core/tests/test_documents.py]: At column 52: (E701) multiple statements on one line (colon)

File: src/core/tests/test_documents.py (hunk 1 above)
Comment (Aareon) [reply src/core/tests/test_documents.py]: Odd how I didn't notice that. Darn mobile. Good eye @freakboy3742

**Comment by tiporan** [comment]
Thanks guys! I will fix all issues raised here then update all files.

**Comment by tiporan** [comment]
> @tiporan Thanks for the contribution! To start with, could you please checkoff the PR checklist above?

Hi @danyeaw! Checked!

**Comment by tiporan** [comment]
implementations:gtk check is still Pending but looking at the details it seems everything is ok. Is there something wrong? Just let me know so I can fix whatever needed. Cheers!

**Comment by freakboy3742** [comment]
@tiporan Looks like the CI system had a problem; I've just restarted the build, which has fixed the problem.

**Pull request merged**